contrib/scripts/env-setup.py

```diff
@@ -78,6 +74,10 @@ def install_vm(version, apt_packages):
   apt_cmd = ['apt-get', 'install', '-y']
   apt_cmd.extend(apt_packages)
 
+  if not is_root:
```

Comment:
I'd just use `sudo` everywhere and simplify things.

Reply:
Turns out Kaggle doesn't even have the `sudo` command so I had to case this.